src/web/components/App.tsx

```diff
@@ -399,6 +399,38 @@ export const App = ({ CAPI, NAV }: Props) => {
         CAPI.config.isSensitive,
     ]);
 
+    // ************************
+    // *   Google Analytics   *
+    // ************************
+    useEffect(() => {
+        // TODO: handle when consent removed
+        onConsentChange((state: any) => {
+            const consentGiven = getConsentFor('google-analytics', state);
+            if (consentGiven) {
+                const gaScript = document.createElement('script');
+                gaScript.id = 'ga-script';
+                gaScript.type = 'text/javascript';
+                gaScript.src = 'https://www.google-analytics.com/analytics.js';
+                document.head.appendChild(gaScript);
+
+                const gaCustomScript = document.createElement('script');
+                gaCustomScript.id = 'ga-custom-script';
+                gaCustomScript.type = 'text/javascript';
+                gaCustomScript.src = window.guardian.gaPath;
+                document.head.appendChild(gaCustomScript);
```

Comment:
I've just remembered that there is a `lib` function that does this for you! https://github.com/guardian/libs/blob/main/src/loadScript.ts

Reply:
Nice!

I guess if we don't need to delete them then the id is redundant

Reply:
Excellent, we were having type compatibility problems when pulling in @guardian/libs, so we've made our own copy of the `loadScript` function for now. Much nicer!